### ISSUE TITLE
Pristine need events triggers

### ISSUE THREAD

**Issue by francosalcedo** [sha256/Pristine#29]
It would be great if there are events that are fired when the fields or the form have been validated without the need to call the valid() method.

**Comment by sha256** [comment]
Good idea. Better if we could support traditional event style. Any PR will be appreciated!
